stock_lot_suffix/models/account_analytic_account.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountAnalyticAccount(models.Model):
+    _inherit = "account.analytic.account"
+
+    analytic_category = fields.Char()
```

Comment:
The name is obscure.  I think we should name it as `lot_suffix` or something IMO.